Supply --install-layout=deb install argument, to install files into /usr
instead of /usr/local/. Rename dist-packages/ to site-packages/ again, so
that packages which expect setup.py to install to site-packages/ and depend
on that in their dh_install files don't break. dh_py* will take care of the
final renaming.

--- ../debian/1/class/python-distutils.mk.in	2009-04-28 10:52:50.000000000 +0200
+++ 1/class/python-distutils.mk.in	2009-06-02 17:04:08.000000000 +0200
@@ -38,7 +38,7 @@
 DEB_PYTHON_SETUP_CMD = setup.py
 DEB_PYTHON_CLEAN_ARGS = -a
 DEB_PYTHON_BUILD_ARGS = --build-base="$(CURDIR)/$(DEB_BUILDDIR)/build"
-DEB_PYTHON_INSTALL_ARGS_ALL = --no-compile -O0
+DEB_PYTHON_INSTALL_ARGS_ALL = --no-compile -O0 --install-layout=deb
 
 #DEB_PYTHON_DESTDIR = $(DEB_DESTDIR)
 
@@ -141,11 +142,23 @@
 # install stage
 $(patsubst %,install/%,$(cdbs_python_indep_packages)) :: install/%: python-install-py
 	cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$(cdbs_python_compile_version)) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) $(DEB_PYTHON_INSTALL_ARGS_ALL)
+	for ddir in $(cdbs_python_destdir)/usr/lib/python?.?/dist-packages; do \
+	  sdir=$$(dirname $$ddir)/site-packages; \
+	  mkdir -p $$sdir; \
+	  tar -c -f - -C $$ddir . | tar -x -f - -C $$sdir; \
+	  rm -rf $$ddir; \
+	done
 
 $(patsubst %,install/%,$(cdbs_python_arch_packages)) :: install/%: $(addprefix python-install-, $(cdbs_python_build_versions))
 	for buildver in $(cdbs_python_build_versions); do \
 		cd $(CURDIR) && cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$$buildver) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) $(DEB_PYTHON_INSTALL_ARGS_ALL); \
 	done
+	for ddir in $(cdbs_python_destdir)/usr/lib/python?.?/dist-packages; do \
+	  sdir=$$(dirname $$ddir)/site-packages; \
+	  mkdir -p $$sdir; \
+	  tar -c -f - -C $$ddir . | tar -x -f - -C $$sdir; \
+	  rm -rf $$ddir; \
+	done
 
 # Deprecated targets.  You should use above targets instead.
 $(addprefix python-build-stamp-, $(cdbs_python_build_versions)):
